lib/generators/maestrano/templates/saml_controller.rb

```diff
@@ -25,6 +25,11 @@ def consume
     # user = User.find_or_create_for_maestrano(user_auth_hash)
     # organization = Organization.find_or_create_for_maestrano(group_auth_hash)
     #
+    # user.tenant = params[:tenant]
+    # user.save
+    # organization.tenant = params[:tenant]
+    # organization.save
```

Comment:
Not so pretty.
Another way to do this would be to include the tenant in the user_auth_hash and group_auth_hash (the tenant is not included in the response right now). With this the routes would not need to be customize by tenant.
Another prettier solution would be to change find_or_create_for_maestrano to accept a second parameter tenant.